pkg/mimirtool/commands/testdata/apiserver.json

```diff
@@ -1461,11 +1461,34 @@
             "type": "datasource"
          },
          {
-            "allValue": null,
+            "allvalue": null,
             "current": { },
             "datasource": "$datasource",
             "hide": 2,
-            "includeAll": false,
+            "includeall": false,
+            "label": "manuallycreated",
+            "multi": false,
+            "name": "manuallycreated",
+            "options": [ ],
+            "query": {
+                "query": "label_values(kube_pod_info, cluster)",
+                "refId": "ManuallyCreatedVar"
+            },
+            "refresh": 2,
+            "regex": "",
+            "sort": 1,
+            "tagvaluesquery": "",
```

Comment:
I ran the tests with camel-cased and lowercase JSON and the detection works in both times.

I checked this on Grafana 9.1 and 9.4 and the JSON model seems to be camel-cased. Where did you get the lowercase one?

Reply:
I copied the JSON block that was already in this testdata file and just modified the query field 😬

Reply:
I'm not sure what your merge flow looks like, but I could either fix just the added chunk in this PR and amend the commit and fix the other as a separate commit within this PR (assuming rebase+merge)

Reply:
my bad, must not be getting enough sleep 🙈  assuming my editor went rouge and I didn't notice.

Just pushed a fix, but let me squash that down